scipy/stats/tests/test_kdeoth.py

```diff
@@ -1,4 +1,4 @@
-from scipy import stats
+from scipy import stats, linalg
```

Comment:
ModuleNotFoundError: No module named 'scipy' the same error after all try

Reply:
I'm not sure what you are trying to say, but I think it has more to do with how you tried to build or install scipy than the changes in this PR.